### ISSUE TITLE
blueprint error handler not working??

### ISSUE THREAD

**Issue by tyan4g** [pallets/flask#1935]
follow the guide on doc
http://flask.pocoo.org/docs/0.11/blueprints/

```
@simple_page.errorhandler(404)
def page_not_found(e):
    return jsonify(result=result)
```

flask version is 0.11.1
still render the offcial 404 page.

**Comment by ThiefMaster** [comment]
Blueprint-level 404 handlers are not possible. A blueprint does not "own" a certain URL space (like `/foo/`) so there is no way to know which blueprint to use to handle the 404 error.

**Comment by tyan4g** [comment]
@ThiefMaster, so the doc is a bit misguiding. I guess i have to fall back to the old way of 
</path:invalid>

**Comment by davidism** [comment]
closed by #1942
